src/python/grpcio/grpc/_channel.py

```diff
@@ -212,7 +213,17 @@ def consume_request_iterator():
         )
         call.start_batch(cygrpc.Operations(operations), event_handler)
         state.due.add(cygrpc.OperationType.send_close_from_client)
-  thread = threading.Thread(target=consume_request_iterator)
+
+  def stop_write_thread(timeout):
```

Comment:
While the patient is on the table: the thread both consumes the request iterator and writes requests to the call, but since "consume_and_write" is too long a term, I just went with the consumption aspect. Maintain that consistency with names like `stop_consumption_thread` and `consumption_thread`? Or tell me I'm being too fussy?